learntools/ml-insights/nbs/tut3.ipynb

```diff
@@ -0,0 +1,215 @@
+    "\n",
+    "Like permutation importance, **partial dependence plots are calculated after a model has been fit.**  The model is fit on real data that has not been artificially manipulated in any way.  \n",
+    "\n",
+    "In our soccer example, teams may differ in many ways. How many passes they made, shots they took, goals they scored, etc. At first glance, it may seem difficult to disentangle the effect of these features. Because the teams that made many passes may also take more shots and score more goals in most cases. \n",
+    "\n",
+    "To see how partial plots separate out the effect of each featuer, we start by considering a single row of data. For example, that row of data might represent a team that had the ball 50% of the time, made 100 passes, took 10 shots and scored 1 goal.\n",
+    "\n",
+    "We will use the fitted model to predict our outcome (probability their player won \"man of the game\". But we **repeatedly alter the value for one variable** to make a series of predictions.  We could predict the outcome if the team had the ball only 40% of the time. We then predict with them having the ball 50% of the time.  Then predict again for 60%.  And so on.  We trace out predicted outcomes (on the vertical axis) as we move from small values of ball possession to large values (on the horizontal axis).\n",
+    "\n",
+    "In this description, we used only a single row of data.  Interactions between features may cause the plot for a single team to be atypical.  So, we repeat that mental experiment with multiple teams from the original dataset, and we plot the average predicted outcome on the vertical axis.  \n",
+    "\n",
+    "You'll see some negative numbers.  That doesn't mean the price would sell for a negative price.  Instead it means the prices would have been less than the actual average price for that distance.\n",
+    "\n",
+    "# Code Example\n",
+    "\n",
+    "First, we have build a model as you've already seen. Model building still isn't our focus, so this is very rudimentary."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "from sklearn.model_selection import train_test_split\n",
+    "from sklearn.ensemble import RandomForestClassifier\n",
+    "\n",
+    "data = pd.read_csv('../input/fifa-2018-match-statistics/FIFA 2018 Statistics.csv')\n",
+    "y = (data['Man of the Match'] == \"Yes\")  # Convert from string \"Yes\"/\"No\" to binary\n",
+    "feature_names = [i for i in data.columns if data[i].dtype in [np.int64, np.int64]]\n",
+    "X = data[feature_names]\n",
+    "train_X, val_X, train_y, val_y = train_test_split(X, y, random_state=1)\n",
+    "my_model = RandomForestClassifier(random_state=0).fit(train_X, train_y)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here is the code to create the Partial Dependence Plot."
```

Comment:
Might be good to call out the library you're using at this point (and maybe link to the docs, in case readers want to explore more of its API)